client/client-lib/src/lib.rs

```diff
@@ -384,6 +384,11 @@ impl<T: AsRef<ClientConfig> + Clone> Client<T> {
         Ok(OutPoint { txid, out_idx: 0 })
     }
 
+    /// Receive e-cash directly when connected to fedimint
```

Comment:
Comments always great! :tada: 

But I don't under "directly" and "when connected to fedimint" here. :)

Reply:
Let me reword, it lets you receive e-cash directly from another peer when online (vs. offline transfer)